source/guides/administrator.rst

```diff
@@ -27,32 +27,22 @@ Learn how to get Mattermost running on your environment.
    :glob:
 
    /install/requirements.rst
-   Installing on Ubuntu 16.04 LTS </install/install-ubuntu-1604.rst>
    Installing on Ubuntu 18.04 LTS </install/install-ubuntu-1804.rst>
    Installing on Kubernetes </install/install-kubernetes.rst>
    Installing on Debian Buster </install/install-debian.rst>
-   Installing on RHEL 6 </install/install-rhel-6.rst>
    Installing on RHEL 7 </install/install-rhel-7.rst>
    Installing on CentOS, Oracle Linux, and Scientific Linux </install/install-centos-oracle-scientific.rst>
    Instructions On Setting Up a Socket-based Mattermost Database </install/sockets-db.rst>
    /install/ee-install*
    /install/transport-encryption/config.rst
    /install/deploy-bitnami*
    /install/docker-local*
-   /install/prod-docker*
    /install/docker-ebs*
    /install/deploy-cloudron*
    Installing Mattermost Team Edition in GitLab Helm Chart </install/install-mmte-helm-gitlab-helm.rst>
-   /install/trouble_mysql.rst
```

Comment:
Do we want to remove this?

Reply:
I've moved it into the Troubleshooting section. The style is different (more content, fewer FAQ-style entries) but it seemed like a better fit in terms of subject matter. What are your thoughts?